examples/browser/irma-web/README.md

```diff
@@ -4,17 +4,19 @@ This package is an example for how to use the `irma-web` plugin in the
 web browser. See the source of `index.js` and `public/index.html` for how we use
 the plugin.
 
-You can run this example by running these commands in this directory:
+You can build this example by running the following commands in this directory:
 
 ```bash
 npm install
 npm run build
```

Comment:
Unlike the other example README's, `npm run link # Only necessary for development builds`  is not included here; is that intentional?

Reply:
True, that's because these examples don't use one of the packages that have dependencies that need to be linked. Only the examples that depend on `irma-frontend` or `irma-popup` need linking steps.